src/SFML/Audio/SoundSource.cpp

```diff
@@ -25,6 +25,7 @@
 ////////////////////////////////////////////////////////////
 // Headers
 ////////////////////////////////////////////////////////////
+#include "SFML/System/Angle.hpp"
```

Comment:
```suggestion
```

`SoundSource::Cone` already uses `sf::Angle` to define its data layout so we can guarantee it's included in SoundSource.hpp. I'm guessing something like clangd automatically added this as evidence by the quotes instead of angle brackets.